### ISSUE TITLE
Article truncated on theatlantic.com

### ISSUE THREAD

**Issue by williamsmj** [mozilla/readability#620]
The reader view of https://www.theatlantic.com/ideas/archive/2020/08/californias-disasters-are-a-warning-climate-change-is-here/615610/ ends in the middle of the article, immediately after the "Read: This is what adapting to climate change looks like" pullquote.

**Comment by PalmerAL** [comment]
This was fixed by #589, I think mostly by luck; the page is structured like this:
```
<div>
  <div> article content </div>
   <aside role="complementary">
</div>
<div>
  <div> article content </div>
   <aside role="complementary">
</div>
<div>
  <div> article content </div>
   <aside role="complementary">
</div>
```

And removing the `aside`s makes the candidate selection logic work better.

**Comment by gijsk** [comment]
We need to include this in Firefox, I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1667145 for this.
